src/localplayer.cpp

```diff
@@ -313,7 +313,7 @@ void LocalPlayer::move(f32 dtime, Environment *env, f32 pos_max_d,
 
 			// The node to be sneaked on has to be walkable
 			node = map->getNodeNoEx(p, &is_valid_position);
-			if (!is_valid_position || nodemgr->get(node).walkable == false)
+			if (!is_valid_position || !nodemgr->get(node).walkable)
```

Comment:
in fact no, i was only editing the file and see it, but i can remove it, to ensure only PR comment is covered